teachertool/public/index.html

```diff
@@ -7,17 +7,16 @@
     <meta name="theme-color" content="#000000" />
     <meta
       name="description"
-      content="MakeCode Teacher Tool. Automatically analyze and evaluate projects."
+      content="MakeCode Project Insights. Designed to help teachers evaluate student projects using a rubric."
```

Comment:
FYI, don't need to block on this, but in our sync yesterday, there was some disagreement around the name Project Insights. We probably need to have a meeting with branding folks to hash this out.